core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -383,14 +383,14 @@ static RelDataType deriveTypeSplit(SqlOperatorBinding operatorBinding,
       SUBSTR.withKind(SqlKind.SUBSTR_POSTGRESQL);
 
   /** The "GREATEST(value, value)" function. */
-  @LibraryOperator(libraries = {BIG_QUERY, ORACLE})
+  @LibraryOperator(libraries = {BIG_QUERY, ORACLE, POSTGRESQL})
```

Comment:
could you please improve UT in `SqlOperatorTest`?

Reply:
@zoudan This looks like a small change and I have it here:
https://github.com/dssysolyatin/calcite/pull/1

What is the best way to move this PR forward?